.appveyor.yml

```diff
@@ -168,9 +168,9 @@ environment:
       CODECOV_BINARY: https://uploader.codecov.io/latest/macos/codecov
 
     # Test alternative Python versions
-    - ID: Ubu20P37a
+    - ID: Ubu20P311a
       # ~35min
-      PY: 3.7
+      PY: 3.11
```

Comment:
can we get to 3.8 wherever it was 3.7 here ? the point of using 3.7 was to ensure that code is compatible with the minimal python version we have supported. If we just crank up to 3.11 right away, we might miss some cases of incompatibility etc.

while eu is asleep I will add those changes for 3.8 -- if doesn't work, feel free to drop that commit

Reply:
actually screw it -- I kinda do not see global pattern there... but we now might be testing less with 3.8 than we were with 3.7... who knows ;-) green -- is good!

Reply:
FTR, I think the "main" Ubuntu runs, (`Ubu20core` & `Ubu20a1` which have no python version specified) are on 3.8 - at least they were the last time I looked. So if setting only here, we might have ended up with testing on 3.8 twice @yarikoptic 

I made a note about that in the commit message 90ace3fe1188615bb812b58549178ca1754abedc